dev/bots/cirrus_setup.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+# This script is only meant to be run by the Cirrus CI system, not locally.
+# It must be run from the root of the Flutter repo.
+
+# Collects log output in a tmpfile, but only prints it if the command fails.
```

Comment:
This is to reduce the huge amount of log output we are producing, but not reduce the output in the case where we need to diagnose a failure.

No, you won't see a log unless it fails.  Yeah, it would be better if we could see some progress, but the alternative is MBs of log output, which is more than useless.

Reply:
SGTM